openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/course_outline.html

```diff
@@ -0,0 +1,119 @@
+{% load i18n %}
+
+<div id='course_title' class='h3'></div>
+<hr>
+<div id='outline'></div>
+
+<style>
+  #outline .fa-folder-o {
+    color: var(--bs-gray-500);
+    padding-right: 0.5rem;
+  }
+  .sequential-item {
+    padding-left: 2rem !important;
+  }
+  .vertical-item {
+    padding-left: 3rem !important;
+  }
```

Comment:
@alexjmpb I don't think modifying the border-radius is necessary, also using a smaller font size on a sequential item looks kinda off to me, note that the default body font-size is 1rem. What do you think instead on only changing the padding of each different type of item and using a different icon for vertical items?:

```
#outline .fa {
  margin-right: 0.5rem;
}
#outline .chapter-item {
  padding: 1rem;
}
#outline .sequential-item {
  padding: 0.8rem 2rem;
}
#outline .vertical-item {
  padding-left: 3rem;
}
```

```
// Create icon element.
const icon = Object.assign(document.createElement('i'), {
  className: `fa ${block.type != 'vertical' ? 'fa-folder-o' : 'fa-file-o'}`,
  ariaHidden: 'true',
});
```

Reply:
I think it would be worth to highlight at least the chapters, either by increasing the font size or making it bolder. Respecting changing the icon for the units, I agree.